engine/voxel_engine_gd.cpp

```diff
@@ -177,6 +177,16 @@ Dictionary VoxelEngine::get_stats() const {
 	return to_dict(zylann::voxel::VoxelEngine::get_singleton().get_stats());
 }
 
+int VoxelEngine::get_thread_count() const {
+	return zylann::voxel::VoxelEngine::get_singleton().get_thread_count();
+}
+
+void VoxelEngine::set_thread_count(uint32_t count) {
```

Comment:
Don't change the exposed method to use `uint32_t`, keep it as `int` and check if negative.